themes/ddbasic/sass/components/node/news.scss

```diff
@@ -492,7 +492,7 @@
   &.alternative-layout-full {
     .ding-news-right {
       float: right;
-      margin-right: getColumn(2) + getGutter(12);
+      margin-right: 8%;
```

Comment:
This change replaces a computed value related to the setup of the layout with a fixed - although percentage-based value.

Have you checked whether is possible to use the existing setup to generate the correct value?

Reply:
I could not get the exact value for the width that the PM described. But I have now changed it to use the getColumn mixin since we prefer this approach, and I don't think the PM will notice the small width difference.